### ISSUE TITLE
Bad PSNR=27.7

### ISSUE THREAD

**Issue by chwangteng** [avinashpaliwal/Super-SloMo#33]
My PSNR=27.7

If I download 300K(as the paper says) frames from high-quality 240fps videos from youtube, will the PSNR rise? And if I use 1080p or 4K videos for trianing, will the result be even better? It takes several days to train on Titan Xp.

**Comment by avinashpaliwal** [comment]
Which dataset are you calculating this PSNR on? The results I have mentioned are calculated on the UCF101 test set. I think a larger dataset will improve performance but it depends on the kind of videos you have. It does take long. Maybe the dataloader or preprocessing can be optimized further.

**Comment by chwangteng** [comment]
My PSNR is calculated during the training process, which is shown in the tensorboard. I will calculate 
 on the UCF101 later, it's my fault. 
I converted a 30fps 12s video and set --sf 30 --fps240, I got a 240fps 44s video, what's wrong with it, why did the duration bacame 44s?

**Comment by chwangteng** [comment]
The output video  stutters every few seconds, do you have this issue? I have tried different resolutiosn and bitrates.
